test/sys/test_aio.rs

```diff
@@ -26,6 +26,7 @@ fn poll_aio(mut aiocb: &mut AioCb) -> Result<()> {
 // bindings.  So it's sufficient to check that AioCb.cancel returned any
 // AioCancelStat value.
 #[test]
+#[cfg_attr(all(target_env = "musl", target_arch = "x86_64"), ignore)]
```

Comment:
You bring up a good point. i don't think `cross` runs things in QEMU based on [this line](https://github.com/japaric/cross/blob/master/src/main.rs#L227).